src/TheNextWeek/main.cc

```diff
@@ -155,33 +154,33 @@ void earth() {
     cam.max_depth         = 50;
     cam.background        = color(0.70, 0.80, 1.00);
 
-    cam.lookfrom = point3(0,0,12);
+    cam.lookfrom = point3(13,2,3);
     cam.lookat   = point3(0,0,0);
     cam.vup      = vec3(0,1,0);
-    cam.vfov     = 20.0;
+    cam.vfov     = 20;
 
-    cam.aperture = 0.0;
+    cam.aperture = 0;
 
-    cam.render(hittable_list(globe));
+    cam.render(world);
 }
 
 
 void quads() {
     hittable_list world;
 
     // Materials
-    auto left_red = make_shared<lambertian>(color(1,.2,.2));
-    auto back_green = make_shared<lambertian>(color(.2,1,.2));
-    auto right_blue = make_shared<lambertian>(color(.2,.2,1));
-    auto upper_orange = make_shared<lambertian>(color(1,.5,0));
-    auto lower_teal = make_shared<lambertian>(color(.2,.8,.8));
+    auto left_red     = make_shared<lambertian>(color(1.0, 0.2, 0.2));
+    auto back_green   = make_shared<lambertian>(color(0.2, 1.0, 0.2));
+    auto right_blue   = make_shared<lambertian>(color(0.2, 0.2, 1.0));
+    auto upper_orange = make_shared<lambertian>(color(1.0, 0.5, 0.0));
+    auto lower_teal   = make_shared<lambertian>(color(0.2, 0.8, 0.8));
 
     // Quads
-    world.add(make_shared<quad>(point3(-3,-2,5), vec3(0,0,-4), vec3(0,4,0), left_red));
-    world.add(make_shared<quad>(point3(-2,-2,0), vec3(4,0,0), vec3(0,4,0), back_green));
-    world.add(make_shared<quad>(point3(3,-2,1), vec3(0,0,4), vec3(0,4,0), right_blue));
-    world.add(make_shared<quad>(point3(-2,3,1), vec3(4,0,0), vec3(0,0,4), upper_orange));
-    world.add(make_shared<quad>(point3(-2,-3,5), vec3(4,0,0), vec3(0,0,-4), lower_teal));
+    world.add(make_shared<quad>(point3(-3,-2, 5), vec3(0,0,-4), vec3(0,4,0),  left_red));
+    world.add(make_shared<quad>(point3(-2,-2, 0), vec3(4,0,0),  vec3(0,4,0),  back_green));
+    world.add(make_shared<quad>(point3( 3,-2, 1), vec3(0,0,4),  vec3(0,4,0),  right_blue));
+    world.add(make_shared<quad>(point3(-2, 3, 1), vec3(4,0,0),  vec3(0,0,4),  upper_orange));
+    world.add(make_shared<quad>(point3(-2,-3, 5), vec3(4,0,0),  vec3(0,0,-4), lower_teal));
```

Comment:
The spacing here is a bit uneven.
See `vec3(0,0,-4)` versus `vec3(0,0,4)`

It's FINE. But it caught my eye. Perhaps something to sanitize in rel. Wholly unnecessary but perhaps worth doing

Reply:
Done.